integration/attestation.go

```diff
@@ -45,8 +45,9 @@ func getAttestations(t *testing.T, images []string, attestationAuthority *kritis
 			m[i] = true
 		}
 		for _, o := range occs {
-			if err := remote.DeleteOccurrence(o.OccID); err != nil {
-				t.Logf("could not delete attestations occurrence %s due to %v", o.OccID, err)
+			// TODO(acamadeo): This is hard-coded to work for PGPAttestation. Undo this
+			if err := remote.DeleteOccurrence(o.PGPAttestation.OccID); err != nil {
```

Comment:
Why this change?  It seems to me this file should stay as is.

Reply:
Apologies, the naming was outdated. `occs` stores RawAttestations, not Occurrences. And `o` is a RawAttestation. I adjusted the names accordingly.